BLE Indoor Positioning/src/test/java/com/nexenio/bleindoorpositioning/location/LocationTest.java

```diff
@@ -39,4 +38,18 @@ public void getAngleTo_validLocations_correctAngles() throws Exception {
         assertEquals(350, angle, 5);
     }
 
+    @Test
+    public void getDistanceTo_validLocations_correctDistance() throws Exception {
+        int distance = (int) NEW_YORK_CITY.getDistanceTo(BERLIN);
+        assertEquals(distance, DISTANCE_NYC_BERLIN);
```

Comment:
Change to `assertEquals(DISTANCE_NYC_BERLIN, distance);`. `assert`parameters are always `expected, actual`. Also don't cast to `int`, specify a tolerable `delta` instead.